drizzlepac/devutils/test_hla_flag_filter.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
+"""script to run hla_flag_filter and compare_sourcelists to test hla_flag_filter parameter updates"""
```

Comment:
The name of this module starting with 'test_' is a little misleading, as it is not a pytest module like we have in the 'tests' directory.  It may be better to rename it to something like 'validate_' or 'check_' instead to avoid confusion (by people or by pytest's test discovery code).

Reply:
Issue resolved by commit [87d482a](https://github.com/spacetelescope/drizzlepac/pull/602/commits/87d482ab196d0f32c99277f7550ae4b11d313694)